CMakePresets.json

```diff
@@ -61,8 +64,10 @@
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "RelWithDebInfo",
         "FOAMADAPTER_BUILD_TESTS": "ON",
+        "NEON_BUILD_TESTS": "ON",
```

Comment:
IMO this shouldn't be the default. We don't even enable tests for NEON profiling build.

```suggestion
```